settings.gradle.kts

```diff
@@ -567,6 +567,7 @@ include(":instrumentation:vibur-dbcp-11.0:library")
 include(":instrumentation:vibur-dbcp-11.0:testing")
 include(":instrumentation:wicket-8.0:javaagent")
 include(":instrumentation:zio:zio-2.0:javaagent")
+include(":instrumentation:mybatis:javaagent")
```

Comment:
list of instrumentation modules is alphabetical

Reply:
> list of instrumentation modules is alphabetical

I adjusted it.